api/src/main/java/org/openmrs/module/reporting/cohort/definition/library/BuiltInCohortDefinitionLibrary.java

```diff
@@ -120,7 +121,7 @@ public CohortDefinition getCodedObsSearchAdvanced() {
     }
 
     @DocumentedDefinition("encounterSearchAdvanced")
-    public CohortDefinition getAnyEncounterDuringPeriodWithOccurrence() {
+    public CohortDefinition getEncounterSearchAdvanced() {
         EncounterCohortDefinition cd = new EncounterCohortDefinition();
         cd.addParameter(new Parameter("onOrAfter", "reporting.parameter.startDate", Date.class));
         cd.addParameter(new Parameter("onOrBefore", "reporting.parameter.endDate", Date.class));
```

Comment:
Please label these reporting.parameter.onOrAfter and reporting.parameter.onOrBefore